helpers/fixup-permissions-vendor.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Add read permission for some binaries under system mount
+
+if [ "$TREE_MOUNT" == "" ]; then
+    echo "TREE_MOUNT is unset!"
```

Comment:
Same here.

Reply:
Fixed all bashisms (even if top of scripts reads #!/bin/bash, it can ease portability)